src/usr/local/www/interfaces.php

```diff
@@ -1895,6 +1900,13 @@ function build_gatewayv6_list() {
 	'V6'
 ))->addMask('subnetv6', $pconfig['subnetv6'], 128);
 
+$section->addInput(new Form_Checkbox(
+	'ipv6usev4iface',
+	'Use IPv4 connectivity as parent interface',
+	'IPV6 will use  the IPv4 connectivity link (PPPoE)',
```

Comment:
Change "IPV6" to "IPv6", and there are two spaces between "use" and "the" instead of just one.

Otherwise seems OK.